internal/types/wait_for.go

```diff
@@ -2,9 +2,15 @@ package types
 
 type WaitFor struct {
 	Field []WaitForField
+	Condition []WaitForStatusCondition
 }
 type WaitForField struct {
 	Key       string
 	Value     string
 	ValueType string `mapstructure:"value_type"`
 }
+
+type WaitForStatusCondition struct {
+	Type   string
+	Status string
+}
```

Comment:
@jkabonickAtOlo you might want to fix this before it gets merged.

Reply:
@stevehipwell I might be missing something, what is the problem with the struct?

Reply:
Missing/removed EOF (shows up online but I'm not sure about mobile).

Reply:
I will take care of that. Thank you for pointing it out.